fuzzing/private/binary.bzl

```diff
@@ -36,6 +36,7 @@ Provider for storing information about a fuzz test binary.
         "corpus_dir": "The directory of the corpus files used as input seeds.",
         "dictionary_file": "The dictionary file to use in fuzzing runs.",
         "engine_info": "The `FuzzingEngineInfo` provider of the fuzzing engine used in the fuzz test.",
+        "workspace_name": "The name of the workspace of the fuzz test executable.",
```

Comment:
I'm curious why this field is actually needed. According to the [`File.short_path` documentation](https://docs.bazel.build/versions/master/skylark/lib/File.html#short_path), `short_path` should be sufficient to determine the runfile location.

Reply:
While `short_path` is sufficient to *use* the runfiles tree, the information contained in it does not suffice to *create* it: Since the `short_path`s aren't relative to the runfiles tree root, but the subdirectory corresponding to the workspace of the target, we cannot learn the name of the workspace from any `short_path`. We don't really need the workspace name as any other unique name (distinct from all external workspaces so that there are no collisions) would do, but deterministically deriving such a name just seems more difficult than passing through the real one.